### DISCUSSION TITLE
Is there a remove or delete function?

### DISCUSSION THREAD

**Discussion by mstaroste** [spatie/laravel-schemaless-attributes#91] [Ideas]
Hello,
is there a function to remove a created attribute?
Thanks

**Comment by freekmurze** [comment]
Currently we don't have a function for this. Feel free to PR it with tests.

**Comment by chinleung** [comment]
> Currently we don't have a function for this. Feel free to PR it with tests.

@freekmurze Isn't the `forget` function removing a created attribute?

**Comment by ncatanchin** [comment]
It doesn't seem to work all the time. I don't think it allows to delete an attribute that has child attributes?
